projects/1.20/assets/tofucraftreload/tofucraft/lang/zh_cn.json

```
@@ -0,0 +1,626 @@
+  "container.tofucraft.tf_storage.missing_item_tooltip" : "添加豆腐转变成豆腐能,或添加豆腐盔甲或豆腐工具进行修复",
+  "tooltip.tofucraft.energy" : "豆腐能: %d/%d",
+  "tooltip.tofucraft.tofuforce.broken" : "破损",
+  "death.attack.tofucraft.zunda" : "%1$s 受到 %2$s 的 %3$s 自然能量轰杀",
+  "death.attack.tofucraft.fallingTofu" : "%1$s被%2$s的%3$s陷入困境",
+  "death.attack.tofucraft.soy_spore" : "%1$s受到%2$s的豆浆孢子的影响",
+  "item.tofucraft.tofu_gauntlet.mode" : "模式",
+  "item.tofucraft.tofu_gauntlet.soyshot" : "发射大豆",
+  "item.tofucraft.tofu_gauntlet.homing_shot" : "制导发射",
+  "biome.tofucraft.tofu_waste" : "豆腐荒野",
+  "biome.tofucraft.tofu_plains" : "豆腐草原",
+  "biome.tofucraft.tofu_forest" : "豆腐森林",
+  "biome.tofucraft.tofu_mountain" : "豆腐山脉",
+  "biome.tofucraft.zunda_forest" : "毛豆豆腐森林",
+  "biome.tofucraft.soybean_forest" : "大豆森林",
+  "biome.tofucraft.soybean_forest_sparse" : "稀疏的大豆森林",
+  "biome.tofucraft.tofu_ocean": "豆腐海洋",
+  "biome.tofucraft.tofu_river": "豆腐河流",
+  "biome.tofucraft.tofu_beach": "豆腐沙滩",
+  "fluid.tofucraft.soymilk": "豆奶",
+  "fluid.tofucraft.soymilk_hell": "地狱豆奶",
+  "fluid.tofucraft.soymilk_soul": "灵魂豆奶",
+  "fluid.tofucraft.bittern": "盐卤",
+  "fluid_type.tofucraft.soymilk": "豆奶",
+  "fluid_type.tofucraft.soymilk_hell": "地狱豆奶",
+  "fluid_type.tofucraft.soymilk_soul": "灵魂豆奶",
+  "fluid_type.tofucraft.bittern": "盐卤",
+  "subtitles.entity.tofunian.idle": "豆腐村民：喃喃自语",
+  "subtitles.entity.tofunian.yes": "豆腐村民：赞同",
+  "subtitles.entity.tofunian.no": "豆腐村民：反对",
+  "subtitles.entity.tofunian.hurt": "豆腐村民：受伤",
+  "subtitles.entity.tofunian.death": "豆腐村民：死亡",
+  "subtitles.entity.tofuspider.idle": "豆腐蜘蛛：低吼",
+  "subtitles.entity.tofuspider.hurt": "豆腐蜘蛛：受伤",
+  "subtitles.entity.tofuspider.death": "豆腐蜘蛛：死亡",
+  "subtitles.entity.tofuspider.spit": "豆腐蜘蛛：发怒",
+  "subtitles.entity.zombie_tofunian.idle" : "僵尸豆腐村民：低吼",
+  "subtitles.entity.zombie_tofunian.hurt" : "僵尸豆腐村民：受伤",
+  "subtitles.entity.zombie_tofunian.death" : "僵尸豆腐村民：死亡",
+  "subtitles.block.soybean_soul.soul_breath" : "灵魂的呼吸",
+  "subtitles.block.soybean.soybean_crack" : "大豆破碎",
+  "tofucraft.bittern_bottle.jei_desc" : "可将豆浆凝固获得豆腐。通过对着液体单击右键来使用。",
+  "tofucraft.blocktofukinu.jei_desc" : "这个方块是用盐卤瓶对准豆浆液体方块右键获得的。",
+  "tofucraft.blocktofumomen.jei_desc" : "这个方块是在嫩豆腐上下各放置各种石头，被压制的更硬获得的。",
+  "tofucraft.soymilk.jei_desc" : "每次饮用豆奶都可以获得一些正面效果. 如果你每天坚持饮用,你的体质将会得到巨大改善.",
+  "item.tofucraft.soymilk.drink_day" : "喝了第 %1$s 天份的豆奶",
+  "entity.tofucraft.slabfish.type.bagu_chan" : "bagu_chan",
+  "entity.tofucraft.slabfish.type.tofu" : "豆腐",
+  "tofucraft.tofu_book.tooltip" : "豆腐知识大全 2022",
+  "tofucraft.tofu_book.tooltip2" : "如果你想阅读这本书,请安装Patchouli(帕秋莉手册)mod!",
+  "item.tofucraft.tofu_book" : "豆腐匠人指南",
+  "tofucraft.jei.bittern" : "卤制",
+  "tofucraft.jei.harden" : "压制",
+  "tofucraft.need_defeat_boss" : "首先需要打败Boss",
+  "block.tofucraft.rice_block" : "大米包装",
+  "tofucraft.config.travelerTofunianSpawn" : "豆腐游商刷新",
+  "itemGroup.tofucraft.tofus.main_tab" : "豆腐",
+  "itemGroup.tofucraft.tofu_foods.main_tab" : "豆腐工艺:食物",

```

请仔细阅读《[翻译规范与指南](https://cfpa.site/TransRules/)》，这些相关规范与资料库在我们的readme和CONTRIBUTING里都有介绍

对于创造模式物品栏分栏，我们一般使用`丨`（汉字，gun3）进行分割
如果没有特殊情况，此类问题请全文通查

```suggestion
  "itemGroup.tofucraft.tofu_foods.main_tab" : "豆腐工艺丨食物",
```